.gitignore

```diff
@@ -1,72 +0,0 @@
-reindex
```

Comment:
Please do not change unrelated files. What's the purpose of removing this file here?